### ISSUE TITLE
Discordo crashed - Index out of range

### ISSUE THREAD

**Issue by Charles-Schleich** [rigormorrtiss/discordo#338]
Hello, Awesome App, helps a lot with distractions.

Im sharing a crash i experienced, maybe it helps if other people are experiencing the same

```bash
$ discordo
panic: runtime error: index out of range [-1] [recovered]
	panic: runtime error: index out of range [-1]

goroutine 1 [running]:
github.com/rivo/tview.(*Application).Run.func1()
	/home/charles/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231126152417-33a1d271f2b6/application.go:251 +0x4d
panic({0x92f940, 0xc000d01fc8})
	/usr/lib/go-1.18/src/runtime/panic.go:838 +0x207
github.com/rivo/tview.(*TextView).MouseHandler.func1(0x1?, 0xc000697b10?, 0x77c025?)
	/home/charles/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231126152417-33a1d271f2b6/textview.go:1358 +0x26d
github.com/rivo/tview.(*Box).WrapMouseHandler.func1(0x2160?, 0x10?, 0xc000697b58?)
	/home/charles/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231126152417-33a1d271f2b6/box.go:209 +0x53
github.com/rivo/tview.(*Flex).MouseHandler.func1(0x1?, 0xc000ca3aa0, 0x754ebf?)
	/home/charles/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231126152417-33a1d271f2b6/flex.go:239 +0xd3
github.com/rivo/tview.(*Box).WrapMouseHandler.func1(0x71a0?, 0x10?, 0x7f93c3ff6470?)
	/home/charles/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231126152417-33a1d271f2b6/box.go:209 +0x53
github.com/rivo/tview.(*Flex).MouseHandler.func1(0x7c20?, 0xc000ca3aa0, 0x10?)
	/home/charles/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231126152417-33a1d271f2b6/flex.go:239 +0xd3
github.com/rivo/tview.(*Box).WrapMouseHandler.func1(0x70b0?, 0xc000697d78?, 0x447ccf?)
	/home/charles/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231126152417-33a1d271f2b6/box.go:209 +0x53
github.com/rivo/tview.(*Application).fireMouseActions.func1(0x6500?)
	/home/charles/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231126152417-33a1d271f2b6/application.go:438 +0x182
github.com/rivo/tview.(*Application).fireMouseActions(0xc0001f00e0, 0xc000ca3aa0)
	/home/charles/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231126152417-33a1d271f2b6/application.go:475 +0x2ee
github.com/rivo/tview.(*Application).Run(0xc0001f00e0)
	/home/charles/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231126152417-33a1d271f2b6/application.go:374 +0x39a
github.com/ayn2op/discordo/cmd.Run({0xc000032460, 0x46})
	/home/charles/code/discordo/cmd/run.go:65 +0x151
main.main()
	/home/charles/code/discordo/main.go:17 +0xcd

```

**Comment by b2ag** [comment]
Looks like a bug in a library (tview) to me. Is it still happening? If so, can you give some more information on the mouse state when this bug occurs? Movements, clicking, something like that?

**Comment by ayn2op** [comment]
Fixed: #436
